hw22(2)

```diff
@@ -0,0 +1,5 @@
+SELECT purchases.id, purchases.date, user.first_name, user.last_name
```

Comment:
When you join a table, you should use any reference to this table by its name or alias. In your case, you joined `users`, this means that you should reference its name like `users.first_name` and `users.last_name`